packages/components/scss/custom.scss

```diff
@@ -1,7 +1,7 @@
 // Styling overrides for bootstrap
 
 //Make bootstrap functions available for use in overrides
-@import 'bootstrap/scss/_functions.scss';
+@import '../../../node_modules/bootstrap/scss/_functions.scss';
```

Comment:
I'd be _okay_ keeping this one relative (as it's just in one spot), but add a comment why we need relative instead of doing the normal thing with load-path.

Would doing `load-path=node_modules` in the other projects be enough? Then just need the one relative import here. Would like a comment, and possibly a link to an upstream sass issue.

Also you say this doesn't happen on Windows or Mac? What's different with the watches?

Reply:
Even having glob paths to specify would help: https://github.com/sass/dart-sass/issues/616

I don't know, the watch seems limiting!

Reply:
`load-path=node_modules` would work for the others, it just adds 1-2k watchers for 3 different packages. Which isn't that big of a deal compared to it watching 100,000+.

I'm not sure what WSL was doing since it should be running the Linux kernel, but my guess is the FS is a bit different from a normal Linux FS and doesn't use inotify. Mac uses FSEvents through chokidar

Reply:
Okay let me know if you want me to test that out